Source/Particles/Pusher/UpdatePosition.H

```diff
@@ -42,4 +42,39 @@ void UpdatePosition(amrex::ParticleReal& x, amrex::ParticleReal& y, amrex::Parti
     z += uz * inv_gamma * dt;
 }
 
+/** \brief Push the particle's positions over one timestep,
+ *    given the value of its momenta `ux`, `uy`, `uz` */
```

Comment:
Maybe we could be more explicit in the docstring here:
```
More specifically, instead of using the standard leapfrog algorithm
(`x^{n+1} - x^n = dt*u^{n+1/2}/gamma^{n+1/2}`), here we use the implicit version
`x^{n+1} - x^n = dt*(u^{n+1} - u^{n})/(gamma^{n+1} - gamma^n)
see Eq. 17 in Chen, JCP  407 (2020) 109228
```

Reply:
I added comments. For completeness, I added a comment to the explicit version as well.